File: docs/user/en/query/filtering/examples/string/attribute-starts-with.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeStartsWith("code", "garmin")
				),
				require(
					entityFetch(
						attributeContent("code")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/string/attribute-starts-with.cs
EvitaResponse<SealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeStartsWith("code", "garmin")
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code")
        		)
        	)
        )
	)
);